cypress/cypress/e2e/treaty.cy.ts

```diff
@@ -10,7 +10,7 @@ describe('treaty', () => {
         expect(loc.pathname).to.equal("/how-we-do-it");
     });
 
-    cy.get(`h2`).contains("treaty").click();
+    cy.get(`a`).contains("treaty").click();
```

Comment:
Fixed bug, as the existing Cypress test was failing as the `<a>` tag no longer existed and was now a `<h2>` tag.